source/configure/configuration-settings.rst

```diff
@@ -313,15 +313,15 @@ Enable users to open Direct Message channels with
 
 |all-plans| |cloud| |self-hosted|
 
-**Any user on the Mattermost server**: The Direct Messages **More** menu has the option to open a Direct Message channel with any user on the server.
+**Any user on the Mattermost server**: Users can open a direct message with any user on the server, and all users are returned when finding channels.
```

Comment:
> and all users are returned when finding channels

Is that referring to the users appearing in the channel switcher?

Reply:
Yes, it is. I've borrowed language used across the product docs. Though this text may also benefit from a link to the finding channels page too.

Reply:
Very minor feedback:

> and all users are returned when finding channels

I think being able to DM someone (or not) implies finding them in searchy places, so if you wanted to you could omit that.

> open a direct message

could be simplified to "direct message"; open, close, read, edit (etc) is all implied.

Reply:
Thanks, Martin!